Remove the build for aws-sdk-cpp, which we don't need, and pybind, which
comes from EasyBuild
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/data-0.5.1.orig/setup.py b/data-0.5.1/setup.py
index 3222795..2c7c0ac 100644
--- a/data-0.5.1.orig/setup.py
+++ b/data-0.5.1/setup.py
@@ -153,7 +153,6 @@ if __name__ == "__main__":
     if RUN_BUILD_DEP:
         from tools.gen_pyi import gen_pyi
 
-        _check_submodules()
         gen_pyi()
 
     setup(
